### PR TITLE
Donor circle member badge

### PR DESCRIPTION
**Pull request by sunilsabatp**
Handle rendering of donor circle member badge in profile card

- Display the donor circle member badge when `isMember` is `true` and the profile is set to private.
- Updated the profile card layout to accommodate the donor circle badge accordingly.

### CODE REVIEW

**Review by mariahosfeld — COMMENTED** [review]
Looks good to me.

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "@mui/x-date-pickers": "^5.0.15",
     "@netlify/plugin-nextjs": "^4.33.0",
     "@next/bundle-analyzer": "^10.2.3",
-    "@planet-sdk/common": "^0.1.41",
+    "@planet-sdk/common": "^0.1.43",
```
Comment (mohitb35) [inline package.json:60]: Update to 0.1.46 for the latest `user` type

File: public/static/locales/en/profile.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
   "Profile": {
     "myProfile": {
       "memberSince": "Member since {date}",
-      "userDescription": "{bio}"
+      "userDescription": "{bio}",
```
Comment (mohitb35) [inline public/static/locales/en/profile.json:5]: `userDescription` seems unnecessary and can be removed, even though this does not relate to the changes in the PR itself

File: src/features/user/Profile/ProfileCard/ProfileCard.module.scss
Comment (mohitb35) [inline src/features/user/Profile/ProfileCard/ProfileCard.module.scss:1]: Avoid using absolute positioning unless absolutely necessary.

I tried out the styles below, and it seems to work as needed. 

```
.profileBackground {
  background-image: url('../../../../../public/assets/images/profile-bg-image.png');
  width: 100%;
  background-repeat: no-repeat;
  background-size: cover;
  min-height: 8rem;
  position: relative;
  @include mdTabletView {
    flex: 1;
  }
	// changed code below
  display: flex;
  justify-content: center;
  align-items: flex-end;
}

.donorCircleMemberBadge {
  width: max-content;
  display: flex;
  gap: 9px;
	// changed code below
  height: fit-content;
  margin-bottom: 4.5rem;
	// changed code above
  background: rgba(211, 234, 221, 1);
  border-radius: 16px;
  padding: 8px 16px;
  color: $primaryDarkColor;
  font-size: $fontXSmall;
  line-height: 16.34px;
  font-weight: 700;
  svg path {
    fill: $primaryDarkColor;
  }
}
```

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@sunilsabatp Minor changes suggested

**Review by mohitb35 — APPROVED** [review]
LGTM

**Pull request merged**